tutorials/introductory/usage.py

```diff
@@ -141,94 +137,78 @@
+# to automatically create and manage the figures and axes and just do the
+# plotting.  So one can do (OO-style)
 
 x = np.linspace(0, 2, 100)
 
-plt.plot(x, x, label='linear')
-plt.plot(x, x**2, label='quadratic')
-plt.plot(x, x**3, label='cubic')
+# Note that even in the OO-style, we use `.pyplot.figure` to create the figure.
+fig = plt.figure()  # Create a figure.
+ax = fig.add_subplot()  # Add an axes to the figure.
+ax.plot(x, x, label='linear')  # Plot some data on the axes.
+ax.plot(x, x**2, label='quadratic')  # Plot more data on the axes...
+ax.plot(x, x**3, label='cubic')  # ... and some more.
+ax.set_xlabel('x label')  # Add an x-label to the axes.
+ax.set_ylabel('y label')  # Add a y-label to the axes.
+ax.set_title("Simple Plot")  # Add a title to the axes.
+ax.legend()  # Add a legend.
+
+###############################################################################
+# or (pyplot-style)
 
+x = np.linspace(0, 2, 100)
+
+plt.plot(x, x, label='linear')  # Plot some data on the (implicit) axes.
+plt.plot(x, x**2, label='quadratic')  # etc.
+plt.plot(x, x**3, label='cubic')
 plt.xlabel('x label')
 plt.ylabel('y label')
-
 plt.title("Simple Plot")
-
 plt.legend()
 
-plt.show()
-
 ###############################################################################
-# :mod:`pylab` is a convenience module that bulk imports
-# :mod:`matplotlib.pyplot` (for plotting) and :mod:`numpy`
-# (for mathematics and working with arrays) in a single namespace.
-# pylab is deprecated and its use is strongly discouraged because
-# of namespace pollution. Use pyplot instead.
-#
-# For non-interactive plotting it is suggested
-# to use pyplot to create the figures and then the OO interface for
-# plotting.
-#
-# .. _coding_styles:
-#
-# Coding Styles
-# ==================
-#
-# When viewing this documentation and examples, you will find different
-# coding styles and usage patterns. These styles are perfectly valid
-# and have their pros and cons. Just about all of the examples can be
-# converted into another style and achieve the same results.
-# The only caveat is to avoid mixing the coding styles for your own code.
+# (Actually there is a third approach, for the case where you are embedding
+# Matplotlib in a GUI application, which completely drops pyplot, even for
+# figure creation.  We won't discuss it here; see the corresponding section in
+# the gallery for more info.)
+#
+# Matplotlib's documentation and examples use both the OO and the pyplot
+# approaches (which are equally powerful), and you should feel free to use
+# either (however, it is preferrable pick one of them and stick to it,
+# instead of mixing them).  In general, we suggest using the OO-style for
+# non-interactive plotting (in functions and scripts that are intended to be
+# reused as part of a larger project), and restrict pyplot to interactive
+# plotting (e.g., in a Jupyter notebook).
 #
 # .. note::
 #    Developers for matplotlib have to follow a specific style and guidelines.
```

Comment:
Would remove this note. The guide is primarily targeted at new users. We don't have to bother them with developer-related information.

Reply:
done.
slightly reworded to make it less as if OO was unsuitable for interactive use.